Makefile

```diff
@@ -101,31 +101,48 @@ static_checks:
 	./scripts/check-i18n-style.sh
 	./scripts/check_missing_reboot_apps.exs
 
-APPS=$(shell $(SCRIPTS)/find-apps.sh)
+# Allow user-set CASES environment variable
+ifneq ($(CASES),)
+CASES_ARG := --case $(CASES)
+endif
+# use a different name to store the user-set SUITES environment variable
+# because SUITES is going to mutate
+CT_SUITES := $(SUITES)
 
-.PHONY: $(APPS:%=%-ct)
+## example:
+## env SUITES=apps/appname/test/test_SUITE.erl CASES=t_foo make apps/appname-ct
 define gen-app-ct-target
 $1-ct: $(REBAR) merge-config clean-test-cluster-config
+	@export SUITES=$(CT_SUITES)
```

Comment:
I think this is not necessary. If `SUITES` is set by the user, it will be available for `find-suites.sh` as is.